content/dstu2/diagnostic-report.md

```diff
@@ -0,0 +1,26 @@
+---
+title: DiagnosticReport | FHIR DSTU 2 API
+---
+
+# DiagnosticReport
+
+* TOC
+{:toc}
+
+## Search
+
+Search for DiagnosticReports that meet supplied query parameters:
+
+    GET /DiagnosticReport?:parameters
+
+### Parameters
+
+Name | Type | Description
+-----|------|--------------
+`patient`|`string`| The identity of a patient to list reports for. Example: `12345`
+`subject`|`string`| The identity of a patient to list reports for. Example: `12345`
```

Comment:
Agree with Naveen.  Some resources have subject with only the Patient type, so we support the typeless 'subject' as a parameter.  For DiagnosticReport, there are multiple possible types, so the documentation here should have 'subject:Patient'.